File: Kotlin/Kotlin_Code/Kotlin_Code/src/main/kotlin/function/java/Vararg.java
package function.java;

public class Vararg {

    public static void main(String[] args) {
        printNumbers(1, 2, 3);
        printNumbers(1, 2, 3, 4);
    }

    public static void printNumbers(int ... numbers) {
        for (int number : numbers) {
            System.out.print(number + " ");
        }
        System.out.println();
    }
}

File: Kotlin/Kotlin_Code/Kotlin_Code/src/main/kotlin/function/Vararg.kt
package function

fun main() {
    printNumbers(1, 2, 3)
    printNumbers(1, 2, 3, 4)

    val array = intArrayOf(1, 2, 3, 4)
    printNumbers(*array)
}

fun printNumbers(vararg numbers: Int) {
    for (number in numbers) {
        print("${number} ")
    }
    println()
}